docs/ossm/injection/README.md

```diff
@@ -0,0 +1,100 @@
+## Installing the Sidecar
+### Injection
+In order to take advantage of all of Istio’s features, pods in the mesh must be running an Istio sidecar proxy.
+
+Automatic Istio sidecar injection can be enabled via labels on a namespace level or directly for a given pod. Automatic injection injects the proxy configuration at pod creation time using an [admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/).
+
+> **_NOTE:_** Upstream Istio offers also Manual injection using the ` istioctl kube-inject` which is not supported by OpenShift Service Mesh.
```

Comment:
Github supports [fancy alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) for this sort of thing.

Reply:
Nice, thanks.